Foundation/GTMServiceManagement.h

```diff
@@ -33,40 +33,40 @@ GTM_EXTERN_C_BEGIN
 // This reimplements some of the ServiceManagement framework on 10.5.
 // Caller takes ownership of error if necessary.
 
-Boolean GTMSMJobSubmit(CFDictionaryRef job, CFErrorRef *error);
-Boolean GTMSMJobRemove(CFStringRef jobLabel, CFErrorRef *error);
+Boolean GTMSMJobSubmit(CFDictionaryRef job, CFErrorRef *error) __OSX_AVAILABLE_BUT_DEPRECATED_MSG(__MAC_10_6, __MAC_10_10, __IPHONE_3_0, __IPHONE_8_0, "Replace with XPC, or at the very least ServiceManagement.h.");
```

Comment:
Drop the "or at the very least ServiceManagement.h" part since that's what is deprecated.

Reply:
Done